src/Routes/Routes.tsx

```diff
@@ -165,7 +166,7 @@ export const Routes = memo(() => {
         <Layout>
           <Switch>
             {privateRoutesList}
-            <Redirect from='/' to='/trade' />
+            <Redirect from='/' to={isMobile ? '/home' : '/trade'} />
```

Comment:
@reallybeard what's the intent here? The reason why trade is the initial route is because of perf. reasons, dashboard is inherently a performance bottleneck and overfetches tons

Reply:
discussed this one out